### ISSUE TITLE
Question about aarch64

### ISSUE THREAD

**Issue by deniskin82** [itamae-kitchen/mitamae#93]
Hello,
is there a way to compile mitamae for aarch64(Ubuntu 18.04 Arm64 on RPi 4)?

Found some old version of Mitamae
```
root@pi4:~# ./mitamae local test.rb
 INFO : Starting mitamae...
{"machine"=>"aarch64", "name"=>"Linux", "release"=>"5.3.0-1017-raspi2", "version"=>"5.3.0", "version_major"=>"5.3"}
 INFO : Recipe: /home/ubuntu/test.rb
root@pi4:~# cat test.rb
puts node[:kernel]
```

Thanks.

**Comment by k0kubun** [comment]
I can try including aarch64 in binary releases.

What is that `./mitamae` by the way? Did you compile it from source on RPi 4 or is it `mitamae-armhf-linux`?

**Comment by deniskin82** [comment]
Hi, I actually got it from [here](https://github.com/nna774/dark-kuins.net-infra/blob/master/itamae/hocho.yml#L21), an old version.
And it would be great to able to compile `aarch64` from source.

**Comment by k0kubun** [comment]
> I actually got it from here, an old version.

Interesting. Thanks for letting me know that.

I included `mitamae-aarch64-linux` in released binaries from v0.11.2. Could you try that?
